api/translate.py

```diff
@@ -6,10 +6,17 @@ def translate(update, context):
     message = update.message
 
     if not context.args:
-        # blame Udit for this example
-        text = "*Usage:* `/tl {DEST} - {SENTENCE}`\n"\
-               "*Example:* `/tl en - watashi wa anato no suki desu`\n"\
-               "Defaults to `en` if none provided."
+        try:
+            args = message.reply_to_message.text or message.reply_to_message.caption
+            translator = Translator()
+            translated = translator.translate(args, dest='en')
+            text = translated.text
+        except AttributeError:
+            # blame Udit for this example
+            text = "*Usage:* `/tl {DEST} - {SENTENCE}`\n"\
+                   "*Example:* `/tl en - watashi wa anato no suki desu`\n"\
+                   "Reply with `/tl` to a message to translate. Defaults to `en` if none provided."
```

Comment:
This makes it sound like you can provide a language when you reply

Reply:
Could add that, not sure how to right now

Reply:
yea just change make that known to the user 
```suggestion
                   "Defaults to `en` if none provided."
                   "Reply with `/tl` to a message to translate it to english." 
```